### DISCUSSION TITLE
Uploading attachments

### DISCUSSION THREAD

**Discussion by ginkel** [outline/outline#7908] [General] [answered]
Hi there,

I can't rule out that I am simply missing the obvious, but how do I upload an attachment (such as a PDF) to Outline? If that matters, we are using the self-hosted version.

Thanks,
Thilo

**Comment by Jinra** [comment, marked as answer]
type `/` on a new line or click the plus icon on a new line and hit "file attachment"

**Reply by ginkel** [reply]
Thanks, works like a charm!
